app/views/LocationTracking.js

```diff
@@ -38,17 +38,18 @@ import { PARTICIPATE } from '../constants/storage';
 import { GetStoreData, SetStoreData } from '../helpers/General';
 import { checkIntersect } from '../helpers/Intersect';
 import languages from '../locales/languages';
-//import BroadcastingServices from '../services/BroadcastingService';
 import BackgroundTaskServices from '../services/BackgroundTaskService';
 import LocationServices from '../services/LocationService';
 
+const MAYO_COVID_URL = 'https://www.mayoclinic.org/coronavirus-covid-19';
```

Comment:
a constant's perfect for this case.
just one question... is there a likelihood of this constant MAYO_COVID_URL being used outside of locationTracking? 

If yes, then it might not be intuitive to lookup this URL in locationTracking.
I have seen a small URL constants class in our repo . Although I'm not a big fan of having one URL constants class, if there's a larger use for this constant and if it serves the purpose, then it can be moved there.
I think it is called authorities.js (but should be refactored at some point if it grows because it doesn't contain just links to authorities).

Reply:
I like your thinking. If someone else needs it, we can move it outside then, but for now, only this page uses it. What do you think?

Reply:
okay. that sounds perfect. :)